runtime/development/src/xcm.rs

```diff
@@ -164,7 +165,8 @@ impl TakeRevenue for ToTreasury {
 /// executed.
 pub type Barrier = (
 	TakeWeightCredit,
-	xcm_primitives::AllowTopLevelPaidExecutionDescendOriginFirst<Everything>,
+	//todo(nuno): revisit this
+	// xcm_primitives::AllowTopLevelPaidExecutionDescendOriginFirst<Everything>,
```

Comment:
**CLARIFICATION**

I have not seen that in change in the other runtimes. What is the difference between the two?

Reply:
leaving for later